pkgs/applications/video/gnash/default.nix

```diff
@@ -71,10 +70,10 @@ stdenv.mkDerivation rec {
          --without-gconf
          --enable-gui=gtk"
 
-       # In `libmedia', Gnash compiles with "-I$gst_plugins_base/include",
-       # whereas it really needs "-I$gst_plugins_base/include/gstreamer-0.10".
+       # In `libmedia', Gnash compiles with "-I$gst-plugins-base/include",
```

Comment:
Luckily it's a nix variable which we want to get the derivation path from. For a moment I thought this PR was going to be closed :sweat: